templates/buildkite-elastic.yml

```diff
@@ -126,6 +127,11 @@ Parameters:
     Default: t2.nano
     MinLength: 1
 
+  EBSOptimized:
+    Description: Optional - Enabled EBSOptimized for the instance
```

Comment:
@lox @Vektah We noticed it will not enable EBSOptimized for you by default. A good example would be m4 family.